### DISCUSSION TITLE
No funciona en Android 10 ( un LG g8x)

### DISCUSSION THREAD

**Discussion by Ajjajaja12** [skyline-emu/skyline#1662] [Q&A]
Reparenlo xfa

**Comment by yhak22** [comment]
english please

**Reply by Ajjajaja12** [reply]
doesn't work on android 10

**Reply by Crytonics** [reply]
What build you try run?

**Reply by Ajjajaja12** [reply]
i have a lg g8x and i try to run pokemon lest go pikachu power rangers and one piece and neither works

**Reply by Crytonics** [reply]
You can see here what games are working and with build https://github.com/skyline-emu/skyline-games-list/issues

**Reply by Ajjajaja12** [reply]
I already tried most of the games on that list but none of them work because I get graphic errors like the white screen or only red color but the Audio does work and that only happens on Android 10 and I hope they can fix it in a future update

**Reply by Crytonics** [reply]
Try with this build https://github.com/skyline-emu/skyline/actions/runs/2600303874

**Reply by Ajjajaja12** [reply]
I tried it but it doesn't work for me

**Reply by Ajjajaja12** [reply]
will they fix it?

**Comment by yhak22** [comment]
I think ur device is SD855 
It can run very good try latest build or try update ur phone
Otherwise it's driver issue su u must wait for adreno tool to be implemented ;)

**Comment by Ajjajaja12** [comment]
You already tried it and it doesn't work and it doesn't only happen to me
it also happens to some friends with Android 10 and I can't update it
because Android 11 hasn't arrived yet

**Reply by yhak22** [reply]
no bro , android 11 is alrady came to all LG G8 and G8x my frined updated his phone to android 11 so make sure that android 11 is avilable for your phone.
and as i said if that not work then it's driver issue.

**Reply by Ajjajaja12** [reply]
The update has not arrived and what does it mean by driver

**Reply by yhak22** [reply]
GPU Driver

**Reply by Ajjajaja12** [reply]
how is it done

**Reply by Dekuzearou** [reply]
You can change GPU drivers by unlocking your phone and flashing custom drivers or just wait for Adreno tools
